docs/userguide/promote-legacy-workload.md

```diff
@@ -0,0 +1,58 @@
+# Promote legacy workload
+
+If there is a member cluster that has deployed a workload(like Deployment) that is not managed by Karmada, we want to let Karmada to take over this workload directly and not to cause its pod to restart. To achieve this goal, We can use `karmadactl promote` command.
+
+## Example
+
+### For member cluster of `Push` mode
+There is a `nginx` deployment that belong to `default` namespace in member cluster named `cluster1`.
+
+```
+[root@master1]# kubectl get cluster
+NAME       VERSION   MODE   READY   AGE
+cluster1   v1.22.3   Push   True    24d
+```
+
+```
+[root@cluster1]# kubectl get deploy nginx
+NAME    READY   UP-TO-DATE   AVAILABLE   AGE
+nginx   1/1     1            1           66s
+
+[root@cluster1]# kubectl get pod
+NAME                       READY   STATUS      RESTARTS   AGE
+nginx-6799fc88d8-sqjj4     1/1     Running     0          2m12s
+```
+
+We can promote it to Karmada through executing below command in Karmada control plane.
+
+```
+[root@master1]# karmadactl promote deployment nginx -n default -c member1
+Resource "apps/v1, Resource=deployments"(default/nginx) is promoted successfully
+```
+
+The nginx deployment has been adopted by Karmada.
+
+```
+[root@master1]# kubectl get deploy
+NAME    READY   UP-TO-DATE   AVAILABLE   AGE
+nginx   1/1     1            1           7m25s
+```
+
+And the pod created by nginx deployment in member wasn't restarted.
+
+```
+[root@cluster1]# kubectl get pod
+NAME                       READY   STATUS      RESTARTS   AGE
+nginx-6799fc88d8-sqjj4     1/1     Running     0          15m
+```
+
+### For member cluster of `Pull` mode
+Most steps are same as `Push` mode, only the flags of `karmadactl promote` command are different.
+
+```
+karmadactl promote deployment nginx -n default -c cluster1 --cluster-kubeconfig=<CLUSTER_KUBECONFIG_PATH>
+```
+
+For more flags and example about the command, you can use `karmadactl promote --help`.
+
+> Note: Due to the version of resources in Kubernetes is still upgraded, the apiserver of Karmada control plane is different from member clusters possibly. To avoid compatibility issues, we can specify GVK of a resource, such as replacing `deployment` with `deployment.v1.apps`.
```

Comment:
```suggestion
> Note: As the version upgrade of resources in Kubernetes is in progress, the apiserver of Karmada control plane cloud be different from member clusters. To avoid compatibility issues, you can specify the GVK of a resource, such as replacing `deployment` with `deployment.v1.apps`.
```